### PR TITLE
Allow for multistatement migrations in Neo4j

### PR DESCRIPTION
**Pull request by mvid**

### CODE REVIEW

File: database/neo4j/neo4j.go [hunk 1]
```diff
@@ -69,11 +71,14 @@ func (n *Neo4j) Open(url string) (database.Driver, error) {
...
 	uri.User = nil
 	uri.Scheme = "bolt"
+	multi := uri.Query().Get("x-multi-statement")
+	uri.RawQuery = ""
 
 	return WithInstance(&Config{
 		URL:             uri.String(),
 		AuthToken:       authToken,
 		MigrationsLabel: DefaultMigrationsLabel,
+		MultiStatement:  multi == "true",
```
Comment (dhui) [inline database/neo4j/neo4j.go]: Use `strconv.ParseBool()` and handle the error properly

File: database/neo4j/neo4j.go [hunk 2]
```diff
@@ -113,9 +118,30 @@ func (n *Neo4j) Run(migration io.Reader) (err error) {
 		}
 	}()
 
-	if _, err := neo4j.Collect(session.Run(string(body[:]), nil)); err != nil {
-		return err
+	if n.config.MultiStatement {
+		statements := bytes.Split(body, []byte{';'})
```
Comment (dhui) [inline database/neo4j/neo4j.go]: Might as well make the separate an internal package level variable/constant

File: database/neo4j/neo4j.go (hunk 1 above)
Comment (mvid) [reply database/neo4j/neo4j.go]: Done

File: database/neo4j/neo4j.go (hunk 2 above)
Comment (mvid) [reply database/neo4j/neo4j.go]: Pulled into a constant

File: database/neo4j/neo4j.go [hunk 3]
```diff
@@ -71,14 +73,17 @@ func (n *Neo4j) Open(url string) (database.Driver, error) {
 	authToken := neo4j.BasicAuth(uri.User.Username(), password, "")
 	uri.User = nil
 	uri.Scheme = "bolt"
-	multi := uri.Query().Get("x-multi-statement")
+	multi, err := strconv.ParseBool(uri.Query().Get("x-multi-statement"))
```
Comment (dhui) [inline database/neo4j/neo4j.go]: Need to correctly handle `x-multi-statement` not being specified. e.g. an empty string shouldn't error

File: database/neo4j/neo4j.go [hunk 4]
```diff
@@ -119,7 +124,7 @@ func (n *Neo4j) Run(migration io.Reader) (err error) {
 	}()
 
 	if n.config.MultiStatement {
-		statements := bytes.Split(body, []byte{';'})
+		statements := bytes.Split(body, []byte{StatementSeparator})
```
Comment (dhui) [inline database/neo4j/neo4j.go]: Casting to a `[]byte` should be sufficient. Or make the package level constant/variable a `[]byte`

**Pull request merged**